crabs/detection_tracking/inference_model.py

```diff
@@ -65,12 +68,11 @@ def load_trained_model(self) -> torch.nn.Module:
         -------
         torch.nn.Module
         """
-        model = torch.load(
-            self.args.model_dir,
-            map_location=torch.device(self.args.accelerator),
-        )
-        model.eval()
-        return model
+        # Get trained model
```

Comment:
Can we avoid this code repetition? Maybe doing something similar to what we did with optuna.